ppr-ui/src/components/common/UpdatedBadge.vue

```diff
@@ -1,14 +1,16 @@
 <template>
-  <v-chip
-    v-if="hasChanges"
-    id="updated-badge-component"
-    xSmall
-    variant="flat"
-    color="primary"
-    :data-test-id="`${action}-badge`"
-  >
-    <b>{{ action }}</b>
-  </v-chip>
+  <div>
+    <v-chip
+      v-if="hasChanges"
+      id="updated-badge-component"
+      xSmall
+      variant="flat"
+      color="primary"
+      :data-test-id="`${action.toLocaleLowerCase()}-badge`"
+    >
+      <b>{{ action.toUpperCase() }}</b>
+    </v-chip>
+  </div>
```

Comment:
Why add a redundant root layer?

Reply:
To fix this:
![Screenshot 2024-02-23 at 13 53 19](https://github.com/bcgov/ppr/assets/2333290/63e13673-363e-473b-9344-1de3bf55c945)

Reply:
If it helps make it easier to implement than all good, just move the id to the root layer of the component.

Reply:
Updated